packages/chrome/src/elements/sheet/components/Title.spec.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { render } from 'garden-test-utils';
+import { SheetTitle as Title } from './Title';
+
+describe('Sheet.Title', () => {
+  it('passes ref to underlying DOM element', () => {
+    const ref = React.createRef<HTMLDivElement>();
+    const { container } = render(<Title ref={ref} />);
+
+    expect(container.firstChild).toBe(ref.current);
```

Comment:
Not a blocker, but can be written to align more with `testing-library` guiding principles:

```tsx
render(<Title ref={ref}>Zendesk</Title>);

expect(screen.getByText('Zendesk')).toBe(ref.current);
```